### PR TITLE
[16.0] [MIG] server_environment

### PR DESCRIPTION
**Pull request by FrancoMaxime**
#120

### CODE REVIEW

**Review by baimont — APPROVED** [review]

File: server_environment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright Wingo SA
+# Copyright 2018 Camptocamp (https://www.camptocamp.com).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+{
+    "name": "server configuration environment files",
+    "version": "16.0.1.1.1",
```
Comment (rven) [inline server_environment/__manifest__.py]: Bump module version to 16.0.1.0.0

```suggestion
    "version": "16.0.1.0.0",
```

Please follow the guidelines https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0.
This change doesn't have to be done in the pre-commit stuff commit

**Review by rven — CHANGES_REQUESTED** [review]

File: server_environment/__manifest__.py (hunk 1 above)
Comment (sebalix) [reply server_environment/__manifest__.py]: @FrancoMaxime can you address this comment please? The version change should belong to the migration commit indeed.

Comment (FrancoMaxime) [reply server_environment/__manifest__.py]: Yes sorry,  I'm sick, I'll fix it tomorrow

Comment (sebalix) [reply server_environment/__manifest__.py]: It's fine, we are not in a hurry :)
Could you cherry-pick these fixes as well https://github.com/OCA/server-env/pull/126/commits ?
In the meantime, rest well!

Comment (lmignon) [reply server_environment/__manifest__.py]: @sebalix I took the initiative to apply the requested changes to move forward.

**Review by lmignon — APPROVED** [review]
Thank you @FrancoMaxime :+1:  (code review)

**Review by rousseldenis — APPROVED** [review]

**Comment by lmignon** [comment]
@sebalix Can you update your review plz? Thank you!

**Review by sebalix — APPROVED** [review]

**Pull request merged**